README.md

```diff
@@ -87,19 +91,60 @@ python3 setup.py install --user
 
 ## Usage
 
-`oscc-check.py [-hdelv] [-c <channel>] [-V <vehicle>]`
+`oscc-check.py [-hdelv] [-b <bustype>] [-B <bitrate>] [-c <channel>] [-V <vehicle>]`
 
 ### Options
 
 ```bash
--h --help                            Display this information
--d --disable                         Disable modules only, no further checks (overrides enable)
--e --enable                          Enable modules only, no further checks checks
--l --loop                            Repeat all checks, run continuously
--c <channel>, --channel <channel>    Specify CAN channel, defaults to 'can0'
--V <vehicle>, --vehicle <vehcile>    Specify your vehcile, defaults to 'kia_soul_ev'
-                                     (kia_soul_ev / kia_soul_petrol / kia_niro)
--v --version                         Display version information
+Options:
+    -h --help                            Display this information
+    -d --disable                         Disable modules only, no further checks (overrides enable)
+    -e --enable                          Enable modules only, no further checks checks
+    -l --loop                            Repeat all checks, run continuously
+    -b --bustype <bustype>               CAN bus type [default: socketcan_native]
+                                         (for more see https://python-can.readthedocs.io/en/2.1.0/interfaces.html)
+    -B --bitrate <bitrate>               CAN bus bit rate [default: 500000]
```

Comment:
I haven't seen an instance where the bitrate isn't `500000` - is there a concrete scenario where this is different? If not, can we omit this feature until the need presents itself? My thinking is that if we can omit this functionality, that's a bit less code we need to maintain. :)